apps/precursor-chemicals/index.js

```diff
@@ -236,10 +236,13 @@ const steps = {
   },
 
   '/discharging-licence-responsibilities': {
+    continueOnEdit: true,
```

Comment:
I don't think `continueOnEdit: true,` is necessary, as the next step is not dependent on a changed answer on this step

Reply:
Without continueOnEdit, from the summary page it is possible to change the answers and never be able to visit /extra-application-information again